src/platform.ts

```diff
@@ -182,6 +182,10 @@ export class EltakoMiniSafe2Platform implements DynamicPlatformPlugin {
             instance = new EltakoSwitchAccessory(this, existingAccessory);
             break;
           }
+          case 'eltako_fsr14': {
```

Comment:
Could you please merge the switch cases with `eltako_switch` and `eltako_fsr14` instead of duplicating the code?
The same way I did it with `eltako_blind` and `eltako_tf_blind`

Reply:
done